SofaKernel/modules/SofaBaseTopology/EdgeSetTopologyContainer.cpp

```diff
@@ -128,7 +128,7 @@ void EdgeSetTopologyContainer::createEdgesAroundVertexArray()
         const Edge& edge = edges[edgeId];
         // adding edge in the edge shell of both points
         
-        if (edge[0] >= nbPoints || edge[1] >= nbPoints)
+        if (edge[0] >= (unsigned int)(nbPoints) || edge[1] >= (unsigned int)(nbPoints))
```

Comment:
it should be unsigned int(...) or you should get a "old C style cast" warning no ?

Reply:
Try using unsigned(...)